Contracts/categories/v1.0/get-categories/response.json5

```diff
@@ -1,23 +1,28 @@
 //Response Data
-[
-    {
-        "id":"631273",
-        "name":"Electronics"
-    },
-    {
-        "id":"78237",
-        "name":"Home Appliances"
-    },
-    {
-        "id":"62873",
-        "name":"Clothing"
-    },
-    {
-        "id":"531273",
-        "name":"Toys"
-    },
-    {
-        "id":"6482731",
-        "name":"Books"
-    }
-]
\ No newline at end of file
+{
+    "Categories" : [
+        {
+            "id":"631273",
+            "name":"Electronics"
+        },
+        {
+            "id":"78237",
+            "name":"Home Appliances"
+        },
+        {
+            "id":62873",
+            "name":"Clothing"
+        },
+        {
+            "id":"531273",
+            "name":"Toys"
+        },
+        {
+            "id":"6482731",
+            "name":"Books"
+        }
+    ],
+
+    "status code" : "1231414"
+
```

Comment:
Why is this needed?

Reply:
@tavisca-rrai 
What is status on this ?

Reply:
yes, it is updated